account/migrations/0005_customuser_gov_id_prefix_alter_customuser_gov_id_and_more.py

```diff
@@ -19,7 +19,7 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='customuser',
             name='gov_id',
-            field=models.CharField(max_length=8, unique=True, validators=[account.models.validate_gov_id]),
```

Comment:
Did you remove this manually?

Reply:
Yes, I have moved it manually.

Reply:
I would never recommend removing manually) Better unmigrate, fix the field, migrate again, so it autogenerates. There is a chance that, if I do migration from zero, it will give an error.